packages/teams-js/src/public/stageView.ts

```diff
@@ -70,7 +70,6 @@ export namespace stageView {
   }
 
   /**
-   * @hidden
    * Feature is under development
```

Comment:
Should this ("Feature is under development") be removed? The @beta tag would imply that in the public documentation (and no other public beta features have this note)